src/models/Schema.ts

```diff
@@ -115,27 +119,41 @@ export interface IGraphSchema {
   results: IGraphSchemaType[];
 }
 
-export enum AccessLevel {
+export enum AnyTypeAccessLevel {
```

Comment:
why the "any" prefix? I'm not sure what it is supposed to mean here.

Reply:
it's used for the access right of the "any" category for builtin groups. It's the right to create nodes with new categories when schema is partial mode.

Normally the accessLevel is always greater than "none" because "none" is filtered out except that it can be "none" for this particular case.

Reply:
Ok

Reply:
A comment in the code would be welcome :)